### PR TITLE
Introducing new icons pack

### PR DESCRIPTION
**Pull request by carolahp**
New icons are PNGs generated from their corresponding SGVs

### CODE REVIEW

**Comment by MarcusDenker** [comment]
Failing test  SystemSettingsPersistenceTest>>testDefaultImageDoesNotStoreAnySetting


```
Error
Given Collections do not match!  additions : an Array(a ThemeIconsStoredSetting('#appearance#current'))  missing: #()
Stacktrace
TestFailure
Given Collections do not match!
	additions : an Array(a ThemeIconsStoredSetting('#appearance#current'))
	missing: #()
SystemSettingsPersistenceTest(TestAsserter)>>assert:description:resumable:
SystemSettingsPersistenceTest(TestAsserter)>>assert:description:
SystemSettingsPersistenceTest(TestAsserter)>>assertCollection:hasSameElements:
SystemSettingsPersistenceTest>>testDefaultImageDoesNotStoreAnySetting
SystemSettingsPersistenceTest(TestCase)>>performTest
```

**Comment by carolahp** [comment]
The test SystemSettingsPersistenceTest>>testDefaultImageDoesNotStoreAnySetting also fails in a fresh Pharo 11 image. 
I think it is unrelated to this PR

**Comment by MarcusDenker** [comment]
This is odd as the other PRs that the CI check do not have this test failing.

**Comment by carolahp** [comment]
It's maybe a problem in my configuration? I will investigate and come back

**Comment by carolahp** [comment]
Do not merge yet. 
Waiting for icon repository migration to pharo-project

File: src/Polymorph-Widgets/ThemeIcons.class.st [hunk 1]
```diff
@@ -47,7 +47,7 @@ ThemeIcons class >> availablePacks [
 
 { #category : #accessing }
 ThemeIcons class >> baseUrl [
-	^ 'https://github.com/pharo-project/pharo-icon-packs/archive' asUrl
+	^ 'https://github.com/carolahp/pharo-icons/archive' asUrl
```
Comment (astares) [inline src/Polymorph-Widgets/ThemeIcons.class.st]: We should not point to personal repos.

**Comment by carolahp** [comment]
Yes, I commented above not to merge until we migrate the icons repo to
pharo-project

**Pull request merged**